cgroup/cgroup_linux.go

```diff
@@ -45,14 +46,27 @@ func NewManager(dryRun bool, pid uint32, cgroupMount string, log *zap.SugaredLog
 func (cg cgroup) Read(controller, file string) (string, error) {
 	controllerDir := (*cg.manager).Path(controller)
 	content, err := cgroups.ReadFile(controllerDir, file)
-
 	if err != nil {
 		return "", err
 	}
 
 	return strings.TrimSuffix(content, "\n"), nil
 }
 
+func (cg cgroup) ReadCPUSet() (cpuset.CPUSet, error) {
```

Comment:
It's way more easier to have this here than in the caller, feels to me it's cgroup package responsibility to know such things